components/chart/search/textbox.tsx

```diff
@@ -28,28 +29,34 @@ const AutoComplete = styled.div`
   z-index: 1000;
   ul {
     border-radius: 10px;
+    margin-top: 55px;
+    border-bottom: 3px solid #8d8d8d;
     li {
-      color: #fff;
+      color: #000;
       font-size: 13px;
-      background-color: #9b9b9b;
-      border-bottom: 1px solid #8d8d8d;
-      width: 233px;
+      background-color: #F3EDED;
+      width: 448px;
       padding: 14px;
       display: block;
       cursor: pointer;
-      height: 40px;
-      font-weight: 300;
+      height: 35px;
+      font-weight: 400;
```

Comment:
let's make the width same as the text input
![image](https://github.com/Bcrumbs-net/hychart/assets/2771203/9a25847f-a4fc-4b74-ab3b-f6bfa3c11a87)

Also let's remove the space between the text input and the droplist

Reply:
Also small note, can we add search icon to the text input on the left side

Reply:
done